utils/appliance/__init__.py

```diff
@@ -1604,7 +1604,7 @@ def start_evm_service(self, log_callback=None):
         self._evm_service_command('start', expected_exit_code=0, log_callback=log_callback)
 
     @logger_wrap("Restart EVM Service: {}")
-    def restart_evm_service(self, rude=False, log_callback=None):
+    def restart_evm_service(self, rude=False, wait_for_web_ui=False, log_callback=None):
```

Comment:
@jteehan, correct, adding a new parm like this is fine.  @psav and @mfalesni were taking issue to you previously doing a rude=True which seems to have been addressed I believe.  Guys, care to clarify?